docs/sdks/sdk-ref-javascript-v2.md

```diff
@@ -305,7 +330,9 @@ When a user logs out of the publisher's site, make the following call:
 
 After this function is executed, the [getAdvertisingToken()](#getadvertisingtoken-string) function returns `undefined` and the [isLoginRequired()](#isloginrequired-boolean) function returns `true`.
 
-### abort(): void
+### abort(): void <Deprecated3100 />
+
+This function is deprecated and will be removed altogether in June of 2025. Use [disconnect()](#disconnect-void) instead as it performs the same logic as `abort()` plus a more thorough disconnection. 
```

Comment:
Couple minor wording suggestions. Your call.

This function is deprecated and support will be removed completely in June of 2025. Instead, use [disconnect()](#disconnect-void) which does the same thing and includes improved, more thorough disconnection logic.